static/resources/js/members-common.js

```diff
@@ -87,13 +87,14 @@ function addMember(members_table) {
         }
     }
 
-    if($('#first_name').val()=="" || $('#last_name').val()=="" || $('#date_of_birth').val()=="" ||
-         $('select[name=church-role-select]').val()==null) {
-
+    if($('#first_name').val()=="" || $('#last_name').val()=="" || $('select[name=church-role-select]').val()==null){
         $('#required-fields-alert').show();
 
     }
+    else if(dateFormatValidator($('#date_of_birth').val())== "False"){
```

Comment:
once you fix your function so it returns boolean and not string then the check here should be like this:
`else if(!dateFormatValidator($('#date_of_birth').val()))`